features/users/user_dashboard.feature

```diff
@@ -124,3 +124,18 @@ Feature: User dashboard
     And I should not see "Oldest Series"
   Then I should see "Recent bookmarks"
     And I should see "Work 5" within "#user-bookmarks"
+
+  Scenario: Sorting within user or collection filtered results removes fandom filter
+  Given a media exists with name: "TV Shows", canonical: true
+    And a fandom exists with name: "Star Trek", canonical: true
+    And a fandom exists with name: "Star Trek: Discovery", canonical: true
+  When I am logged in as "meatloaf"
```

Comment:
I'd suggest making this "And," since being logged in and posting works is all background/setup for what we're testing, and not the main behavior.